voter-dapp/src/Styles.js

```diff
@@ -0,0 +1,10 @@
+import { makeStyles } from "@material-ui/styles";
```

Comment:
Nice - I like the idea of putting global-ish component styles in a separate file.